### PR TITLE
Add dark themed websites

### PR DESCRIPTION
**Pull request by alfawal**
and normalize some lines.

### CODE REVIEW

File: src/ui/popup/index.tsx [hunk 1]
```diff
@@ -72,16 +72,16 @@ if (DEBUG) {
         try {
```
Comment (Gusted) [inline src/ui/popup/index.tsx:72]: Please revert these lines. Code-style should take of styling issue. Also not related to adding dark themes PR. Don't clutter 1 PR with multiple things.

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -726,6 +726,7 @@ polsatboxgo.pl
 polsatgo.pl
 pony.tube
 pool.pm
+popper.js.org
```
Comment (Gusted) [inline src/config/dark-sites.config:729]: I don't find the start page that dark, quite flashy when you open the site.

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -769,6 +770,7 @@ redditinvestigator.com
 redeclipse.net
 reelgood.com
 refind.com
+regex101.com
```
Comment (Gusted) [inline src/config/dark-sites.config:773]: Not dark-by-default.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dark-sites.config (hunk 2 above)
Comment (alfawal) [reply src/config/dark-sites.config:729]: This is weird, can you provide some efficient ways to check? incognito mode is not enough I guess.

File: src/config/dark-sites.config (hunk 3 above)
Comment (alfawal) [reply src/config/dark-sites.config:773]: same here

File: src/ui/popup/index.tsx (hunk 1 above)
Comment (alfawal) [reply src/ui/popup/index.tsx:72]: Okay, is there a label or something called "Code-style"? should I do these lines there?

**Review by Gusted — APPROVED** [review]

**Pull request merged**